sheet_to_triples/tests/test_xl.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2021 Visual Meaning Ltd
+# This is free software licensed as GPL-3.0-or-later - see COPYING for terms.
+
+"""Unittests for the xl, xls and xlsx modules of sheet-to-triples."""
+
+import unittest
+
+from unittest import mock
+
+from .. import xl, xlsx, xls
+
+
+class TestSheet:
```

Comment:
Because the prefix `Test` is generally used for testcase classes, I prefer to use some other word (generally 'Stub') for this kinds of test-only implementations of an interface.

Reply:
Yes, I've become painfully aware the naming for all of my test structures is wrong. Was intending to do a final pass once they're all merged in to fix up the names and yank out common mock structures into a test utilities module.